sdk/storage/storage-blob/src/generated/src/operations/service.ts

```diff
@@ -436,7 +436,7 @@ const submitBatchOperationSpec: coreHttp.OperationSpec = {
   },
   contentType: "application/xml; charset=utf-8",
   responses: {
-    202: {
+    200: {
```

Comment:
This should be switched back to 202 with a transform

Reply:
Done.